packages/openai/src/RPA/OpenAI.py

```diff
@@ -0,0 +1,241 @@
+        baselib.authorize_to_openai(secrets["key"])
+
+        result = baselib.completion_create(
+            Create a tagline for icecream shop',
+            temperature=0.6,
+        )
+        print(result)
+    """
+    
+    ROBOT_LIBRARY_SCOPE = "Global"
+    ROBOT_LIBRARY_DOC_FORMAT = "REST"
+
+    def __init__(self) -> None:
+        self.logger = logging.getLogger(__name__)
+
+    @keyword
+    def authorize_to_openai(self, api_key: str) -> None:
+        """Keyword for authorize to OpenAI with your API key obtained from your account.
+
+        :param api_key: Your OpenAI API key
+
+        Robot Framework example:
+
+        .. code-block:: robotframework
+
+            ${secrets}   Get Secret   secret_name=OpenAI
+            Authorize To OpenAI   api_key=${secrets}[key] 
+
+        Python example:
+
+        .. code-block:: python
+
+            secrets = Vault().get_secret("OpenAI")
+            baselib = OpenAI()
+            baselib.authorize_to_openai(secrets["key"])
+
+        """
+        openai.api_key = api_key
+
+    @keyword
+    def completion_create(self, 
+        prompt: str,
+        model: Optional[str] = "text-davinci-003",
+        temperature: Optional[int] = 0.7,
+        max_tokens: Optional[int] = 256,  
+        top_p: Optional[int] = 1,   
+        frequency_penalty: Optional[int] = 0,   
+        presence_penalty: Optional[int] = 0,
+        result_format: Optional[str] = "string"      
+    ) -> None:
+        """Keyword for creating text completions in OpenAI. Keyword returns a text string.
+
+        :param prompt: Text submitted to OpenAI for creating natural language.
+
+        :param result_format: Result format (string / json). Return just a string or the default JSON response.
+
+        Robot Framework example:
+
+        .. code-block:: robotframework
+
+            ${response}  Completion Create    
+            ...     Write a tagline for an icecream shop.  
+            ...     temperature=0.6
+            Log     ${response}
+
+        Python example:
+
+        .. code-block:: python
+
+            result = baselib.completion_create(
+                'Create a tagline for icecream shop',
+                temperature=0.6,
+            )
+            print(result)
+
+        """
+        response = openai.Completion.create(
+            model=model,
+            prompt=prompt,
+            temperature=temperature,
+            max_tokens=max_tokens,
+            top_p=top_p,
+            frequency_penalty=frequency_penalty,
+            presence_penalty=presence_penalty
+        )
+        self.logger.info(response)
+        if result_format == "string":
+            text = response['choices'][0]['text'].strip()
+            return text
+        if result_format == "json":
+            return response
+        else:
+            return None
+
+    @keyword
+    def image_create(self, 
+        prompt: str, 
+        size: Optional[str] = "512x512", 
+        n: Optional[int] = 1,
```

Comment:
bad parameter name, maybe `count` or `number` would be better

Reply:
ChatGPT told me that num_images is the best argument name for this :)